### PR TITLE
ES: raise error when accessing points from Elasticsearch accessor

### PR DESCRIPTION
**Pull request by adericbourg**

### CODE REVIEW

File: biggraphite/drivers/elasticsearch.py [hunk 1]
```diff
@@ -302,6 +300,10 @@ def _get_depth_from_components(components):
     return len(components) - 1
 
 
+def _raise_unsupported():
+    raise NotImplementedError("Elasticsearch accessor does not support point operations")
```
Comment (iksaif) [inline biggraphite/drivers/elasticsearch.py]: /write/data/

File: biggraphite/drivers/elasticsearch.py [hunk 2]
```diff
@@ -432,34 +426,19 @@ def insert_points_async(self, metric, datapoints, on_done=None):
...
-        if on_done:
-            on_done(None)
+        _raise_unsupported()
 
     def drop_all_metrics(self, *args, **kwargs):
         """See the real Accessor for a description."""
         super(_ElasticSearchAccessor, self).drop_all_metrics(*args, **kwargs)
         # Drop indices.
-        self._metric_to_points.clear()
         self._name_to_metric.clear()
```
Comment (iksaif) [inline biggraphite/drivers/elasticsearch.py]: _name_to_metric / _directory_names should be removed too, there are leftovers from MemoryAccessor

File: biggraphite/drivers/elasticsearch.py (hunk 1 above)
Comment (adericbourg) [reply biggraphite/drivers/elasticsearch.py]: Done.

File: biggraphite/drivers/elasticsearch.py (hunk 2 above)
Comment (adericbourg) [reply biggraphite/drivers/elasticsearch.py]: Done.

**Pull request merged**